File: Programming Syntax/Python2 [outdated]/Standerd Examples/recurse.py
'''
    Author: 
    demostrates recursion type functions: fibbinacie, factorial
'''

def factorial (i):
    if i>1:
        return i*factorial(i-1)
    else:
        return i

def fib (i):
    if i==0:
        return 0
    elif i==1:
        return 1
    else:
        return fib(i-1) + fib(i-2)
    
if __name__ == "__main__":
    for i in range(0,11): #[0,21)
        print str(i) + "!\t= " + str(factorial(i)) + "\t\tfib(" + str(i) + ")\t= " + str(fib(i))

File: PHP/Examples/recurse.php
<?php
	function factorial ($t1){
		if ($t1 > 0){
			return $t1*factorial($t1-1);
		}
		else {
			return 1;
		}
	}
	
	function fib ($t1){
		if ($t1 == 0){
			return 0;
		}
		elseif ($t1 == 1){
			return 1;
		}
		else {
			return fib($t1 - 1) + fib($t1 - 2);
		}
	}

	for ($i=0;$i<=10;$i++){
		echo "$i!\t= ", factorial($i), "\t\tfib($i)\t= ", fib($i), "<br>\n";
	}
?>